Formula/g/gstreamer.rb

```diff
@@ -117,7 +117,11 @@ def python3
   patch :DATA
 
   def install
+    odie "rs resource needs to be updated" if build.stable? && version != resource("rs").version
+
     (buildpath/"subprojects/gst-plugins-rs").install resource("rs")
+    # fix dav1d version constraint
+    inreplace "subprojects/gst-plugins-rs/meson.build", "['>=1.0', '<1.3'", "['>=1.0', '<1.5'"
```

Comment:
I don't understand this change—dav1d 1.3 has a new API that the gstdav1d plugin is not compatible with, that's why that version restriction exists.

Reply:
I tried building with 1.4, and it builds and works superficially, but will apparently crash randomly at runtime. There is an MR upstream to move to the 1.3 API, that will be needed before this can be enabled: https://gitlab.freedesktop.org/gstreamer/gst-plugins-rs/-/merge_requests/1393

Reply:
We are building with newer Rust `dav1d` version (0.10). No idea if it is working for users as we don't have a test to confirm functionality.